src/Riok.Mapperly/Descriptors/MappingBuilders/StringToEnumMappingBuilder.cs

```diff
@@ -22,23 +24,53 @@ public static class StringToEnumMappingBuilder
             .Any(x => x.IsGenericMethod);
 
         if (ctx.IsExpression)
+        {
```

Comment:
Are this brackets necessary? If yes, then also add below on L54.

Reply:
Fixed.